src/components/pinning-manager/PinningManager.js

```diff
@@ -0,0 +1,151 @@
+import React, { Fragment, useEffect, useState, useMemo, useRef } from 'react'
+import { connect } from 'redux-bundler-react'
+import filesize from 'filesize'
+import { AutoSizer, Table, Column, SortDirection } from 'react-virtualized'
+import { sortByProperty } from '../../lib/sort'
+
+// Components
+// import Button from '../button/Button'
+import Overlay from '../overlay/Overlay'
+import PinningModal from './pinning-manager-modal/PinningManagerModal'
+import GlyphPin from '../../icons/GlyphPin'
+import ContextMenu from '../context-menu/ContextMenu'
+import ContextMenuItem from '../context-menu/ContextMenuItem'
+import GlyphDots from '../../icons/GlyphDots'
+import StrokeSettings from '../../icons/StrokeSettings'
+import StrokeCancel from '../../icons/StrokeCancel'
+import StrokeExternalLink from '../../icons/StrokeExternalLink'
+
+import './PinningManager.css'
+
+const ROW_HEIGHT = 50
+const HEADER_HEIGHT = 32
+
+export const PinningManager = ({ pinningServices, doFilesSizeGet, doFilesFetch, filesSize, t }) => {
+  const [isModalOpen, setModalOpen] = useState(false)
+  // const onModalOpen = () => setModalOpen(true)
+  const onModalClose = () => setModalOpen(false)
+
+  const [sortSettings, setSortSettings] = useState({
+    sortBy: 'addedAt',
+    sortDirection: SortDirection.ASC
+  })
+  useEffect(() => {
+    (async () => {
+      await doFilesFetch()
+      await doFilesSizeGet()
+    })()
+  }, [doFilesFetch, doFilesSizeGet])
+
+  const localPinning = useMemo(() =>
+    ({ name: t('localPinning'), type: 'LOCAL', svgIcon: GlyphPin, totalSize: filesSize }),
+  [filesSize, t])
+
+  const sortedServices = useMemo(() =>
+    (pinningServices || []).sort(sortByProperty(sortSettings.sortBy, sortSettings.sortDirection === SortDirection.ASC ? 1 : -1)),
+  [pinningServices, sortSettings.sortBy, sortSettings.sortDirection])
+
+  const sortedList = useMemo(() => [localPinning, ...sortedServices], [localPinning, sortedServices])
+
+  return (
+    <Fragment>
+      <div className="mv4 pinningManager">
+        <div className='ph4 flex items-center bg-white lh-copy charcoal f6 fw5'>
```

Comment:
```suggestion
        <div className='ph3 flex items-center bg-white lh-copy charcoal f6 fw5'>
```
Looks a little more consistent, and gives us a tiny bit more space at narrower widths.